docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -66,13 +80,42 @@ kubectl apply -f my-medusa-bucket-key.yaml
 secret/medusa-bucket-key configured
 ```
 
-### Create or update the k8ssandra
+### Create or update the k8ssandra cluster
+
+Install the `k8ssandra` chart with the following properties. You can reference the provided [backup-restore-values.yaml](./backup-restore-values.yaml) file. It contains:
+
+```
+size: 3
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    multiTenant: true
+    storage: s3
+```
+
+The chart's entries relate to a Kubernetes Secret, which contains the object store credentials. 
```

Comment:
More specifically the `bucketSecret` property specifies the name of a secret that should contain an AWS access key. The AWS account with which the key is associated should have the permissions as described in the [Medusa docs](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md) that are required for Medusa to access the S3 bucket.

Reply:
Incorporated - thanks John